cardano-node/src/Cardano/Node/Protocol/Byron.hs

```diff
@@ -177,9 +177,14 @@ readLeaderCredentials genesisConfig
                          . hoistEither
                          $ canonicalDecodePretty delegCertFileBytes
 
+          -- The label is used to distinguish multiple sets of credentials
+          -- during tracing. We use the public signing key as the label. TODO
+          -- @deepfire works for you? Or should we hash it first?
+         let label = Text.pack $ show $ Byron.Crypto.toVerification signingKey
```

Comment:
@mrBliss, this looks fine, as we don't have a basis for preference yet.

Reply:
I'm curious what this looks like when traced, I'm afraid that the type doesn't have a user-friendly `Show`..